dashproducts.php

```diff
@@ -197,21 +197,22 @@ public function getTableBestSellers($date_from, $date_to)
 					SELECT
 						product_id,
 						product_name,
-						SUM(product_quantity) as total,
+						SUM(product_quantity-product_quantity_refunded-product_quantity_return-product_quantity_reinjected) as total,
 						p.price as price,
 						pa.price as price_attribute,
 						SUM(total_price_tax_excl / conversion_rate) as sales,
 						SUM(product_quantity * purchase_supplier_price / conversion_rate) as expenses
 					FROM `'._DB_PREFIX_.'orders` o
 		LEFT JOIN `'._DB_PREFIX_.'order_detail` od ON o.id_order = od.id_order
+		LEFT JOIN `' . _DB_PREFIX_ . 'order_state` os ON o.current_state = os.id_order_state
 		LEFT JOIN `'._DB_PREFIX_.'product` p ON p.id_product = product_id
 		LEFT JOIN `'._DB_PREFIX_.'product_attribute` pa ON pa.id_product_attribute = od.product_attribute_id
-		WHERE `invoice_date` BETWEEN "'.pSQL($date_from).' 00:00:00" AND "'.pSQL($date_to).' 23:59:59"
-		AND valid = 1
+		WHERE `date_upd` BETWEEN "'.pSQL($date_from).' 00:00:00" AND "'.pSQL($date_to).' 23:59:59"
+		AND os.logable = 1 AND os.paid = 1
 		'.Shop::addSqlRestriction(false, 'o').'
 		GROUP BY product_id, product_attribute_id
 		ORDER BY total DESC
-		LIMIT '.(int)Configuration::get('DASHPRODUCT_NBR_SHOW_BEST_SELLER', 10)
+		LIMIT '. ((Configuration::get('DASHPRODUCT_NBR_SHOW_BEST_SELLER')) ? (int)Configuration::get('DASHPRODUCT_NBR_SHOW_BEST_SELLER') : 10)
```

Comment:
@Quetzacoalt91 code review please.
Thanks in advance :)

Reply:
The code can be even simpler, because this configuration key is always defined.

On the module installation, it is registered with the default value 10: https://github.com/PrestaShop/dashproducts/blob/7871fe88abc93a0e1622eb3e5150fdd8dda8eefb/dashproducts.php#L52

So, don't check anything, just get the default value.
```sql
LIMIT '.(int)Configuration::get('DASHPRODUCT_NBR_SHOW_BEST_SELLER')
```